### DISCUSSION TITLE
AssertionError: Only the 'gbtree' model type is supported, not 'dart'!

### DISCUSSION THREAD

**Discussion by aniketsnv-1997** [shap/shap#2677] [Q&A]
Hello Everyone

I am trying to get the SHAP Summary plot for an XGBoost model with ```booster=dart``` (came as the value after hyperparameter tuning). However, I am getting this error ```AssertionError: Only the 'gbtree' model type is supported, not 'dart'!```every time I try and generate the SHAP values. Tried to Google it but there are no satisfactory workable solutions out there

Any kind of help will be much appreciated @slundberg

**Comment by georgesabr** [comment]
it's not supported for dart you should use gbtree when setting your model parameters
